graphs/graph_manager.py

```
@@ -0,0 +1,245 @@
+            DiscordError: If posting to Discord fails
+        """
+        try:
+            # Filter enabled graphs to include only actual graph types (exclude settings like ENABLE_GRAPH_GRID)
+            enabled_graphs = {
+                k: v for k, v in self.config.items()
+                if k in self.GRAPH_TYPES and v
+            }
+            logging.debug("Filtered enabled graphs: %s", enabled_graphs.keys())
+
+            # Create pairs of graph types and file paths, ensuring they match exactly
+            graph_types = list(enabled_graphs.keys())
```

Ruff

Local variable `graph_types` is assigned to but never used

Remove assignment to unused variable `graph_types`

(F841)